sdk/src/main/java/io/radar/sdk/Radar.kt

```diff
@@ -1335,6 +1335,23 @@ object Radar {
         RadarSettings.setForegroundService(context, options)
     }
 
+    /**
+     * Settings for the all notification.
+     *
+     * @see [](https://radar.com/documentation/sdk)
+     *
+     * @param[options] Notifications options
+     */
+    @JvmStatic
+    fun setNotificationsOptions(options: RadarNotificationsOptions) {
```

Comment:
let's make `notification` singular — `setNotificationOptions` and `RadarNotificationOptions`